test/e2e/broker_trigger_test.go

```diff
@@ -56,28 +56,17 @@ type eventReceiver struct {
 // and sends different events to the broker's address. Finally, it verifies that only
 // the appropriate events are routed to the subscribers.
 func TestDefaultBrokerWithManyTriggers(t *testing.T) {
-	clients, cleaner := Setup(t, t.Logf)
-
-	// Verify namespace exists.
-	ns, cleanupNS := CreateNamespaceIfNeeded(t, clients, t.Logf)
+	t.Parallel()
 
-	defer cleanupNS()
+	clients, cleaner := Setup(t, t.Logf)
 	defer TearDown(clients, cleaner, t.Logf)
 
-	t.Logf("Labeling namespace %s", ns)
-
-	// Label namespace so that it creates the default broker.
-	err := LabelNamespace(clients, t.Logf, map[string]string{"knative-eventing-injection": "enabled"})
```

Comment:
Similar, to the prior comment. Are we now requiring that the namespace is annotated before calling the go test?

Reply:
Ditto